packages/android_intent/lib/android_intent.dart

```diff
@@ -31,6 +32,7 @@ class AndroidIntent {
       this.category,
```

Comment:
I believe we'd could help `dartfmt` produce something nicer with a trailing comma here:
```dart
const AndroidIntent({
  @required this.action,
  this.category,
  this.data,
  this.arguments,
  this.package,
  Platform platform,
})
```

Reply:
Yes! The more I use Dart the more I'm becoming a trailing comma enthusiast :-)